src/pynwb/_version.py

```diff
@@ -40,8 +40,8 @@ def get_config():
     # _version.py
     cfg = VersioneerConfig()
     cfg.VCS = "git"
-    cfg.style = "pep440-post"
-    cfg.tag_prefix = ""
+    cfg.style = "pep440-pre"
+    cfg.tag_prefix = "'*.*.*'"
```

Comment:
Is it expected to have single quote within double quote ?

Reply:
Nope my bad, fixing it right now.

Reply:
I take it back. It is autogenerated and expected.